### ISSUE TITLE
Variables for notifications

### ISSUE THREAD

**Issue by mneumark** [ccrisan/motioneye#54]
It would be really nice if you could pass variables to notifications, like say, a URL that points to an image taken with the notification, and a URL that points to the recorded video. Perhaps notification time, camera name. Things like that. Stuff you could access from the notification command and the Web Hook Notification.

**Comment by ccrisan** [comment]
The problem is that the `%f` [conversion specifier] (http://www.lavrsen.dk/foswiki/bin/view/Motion/ConversionSpecifiers) is not available with the `on_event_end` directive, which is used for notification. I'll think of a workaround.

P.S. upload to Google Drive and Dropbox are on the way and I'm thinking of adding generic HTTP upload support as well. I believe that's what you're actually looking for.

**Comment by mneumark** [comment]
Actually I was trying to create a push notification system using PushOver. I wanted the push notification in PushOver to contain an image. PuahOver supports a message with HTML so all I need is an image URL and once you do that you may as well add in a couple other variables. Something like:
```bash
curl api.blah.com/push?message=Motion%20detected%20in%20{{camera_name}}&image={{image_url}}
```

**Comment by ccrisan** [comment]
I just added support for webhook calling and command running for each media file created. The feature will be included in the next 0.29 release.

**Comment by mneumark** [comment]
Sweet, that should help.

**Comment by MattMulhern** [comment]
_Sorry for commenting on a closed issue, but there doesn't seem to be a similar open one._

I understand that the "run a command" and "run a webhook"have been added to the media upload section which is awesome. 

Are there any default xargs or environment vars set in the shell running your command specified in the "run a command" dialog? I've got a simple bash script im running and was wondering if there was a handy way to know the actual filepath (just on local disk, not where it's being uploaded to) of the media file being uploaded.
